citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/CitationManagerTest.java

```diff
@@ -12,6 +12,8 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.Mockito;
+import static org.mockito.Mockito.doThrow;
```

Comment:
please use `Mockito.` prefix like all the other calls to mockito to be consistent.

Reply:
?